server/views/content-fragments/additional-needs/index.njk

```diff
@@ -0,0 +1,3 @@
+<section id="san-additional-needs-content-fragment">
```

Comment:
Skelton nunjucks template for the content fragment.
My next PRs will start to flesh this out 👍